openlibrary/core/helpers.py

```diff
@@ -196,7 +212,7 @@ def sprintf(s, *a, **kw):
         return s
 
 
-def cond(pred, true_value, false_value=""):
+def cond(pred, true_value, false_value):
```

Comment:
```suggestion
def cond(pred: Any, true_value: Any, false_value: Any = "") -> Any:
```
Good call on not putting `list[Author]` here, etc., as that could be misleading, given all the possible valid things one could supply here.

My suggestion, using `Any`, is of questionable value, but it least it lets others know the function has been considered vis-a-vis typing.

The main thing is that we'll want to keep the `""` default value for `false_value`, so we don't make `false_value` a required argument for calling functions that may now rely on it having a default.